Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -477,6 +477,8 @@ class SearchViewController: SiteTableViewController,
         let tempSearchQuery = searchQuery
         suggestClient?.query(searchQuery, callback: { suggestions, error in
             if error != nil {
```

Comment:
Could we switch this if around as we are not doing anything with the error anyway? Basically checking `if error == nil`.

Reply:
That's what I intended to do in my original PR I must have just forgot to swap out the operator 🙈